docs/geedocs/5.2.4/answer/3481558.html

```diff
@@ -254,6 +285,106 @@ <h3>Commands</h3>
+  <pre>
+<code><b>gemodifyimageryproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong> [--<strong>historical_imagery</strong> | --<strong>no_historical_imagery</strong>]
+ -o <em>projectname</em> {[--<strong>maxleveloverride</strong> <em>level</em>]}
+ <em>insetresource</em>} </code>...</pre>
+
+  <h3>Purpose</h3>
+  <p>Modifies resources of an existing project.</p>
+  
+  <h3>Commands</h3>
+  <pre><code><b>--mercator </b></code></pre>
+  <p><i>Optional</i>. Uses Mercator map projection for the imagery project.</p>
+  <pre><code><b>--flat </b></code></pre>
+  <p><i>Default</i>. Uses Flat map (Plate Carrée) projection.</p>
+  <pre><code><b>--historical_imagery </b></code></pre>
+  <p><i>Optional</i>. Uses historical imagery for the project.</p>
+  <pre><code><b>--no_historical_imagery </b></code></pre>
+  <p><i>Default</i>. Uses normal imagery for the project.</p>
+  <pre><code><b>--maxleveloverride</b> <i>level</i></code></pre>
+  <p><i>Optional</i>.</p>
+
+
+  <h2>gemodifyterrainproject</h2>
+  <pre>
+<code><b>gemodifyterrainproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong>
+ -o <em>projectname</em> {[--<strong>maxleveloverride</strong> <em>level</em>]}
+ [--<strong>no_terrain_overlay</strong> | {--<strong>terrain_overlay</strong> --<strong>start_level</strong> <em>level</em> --<strong>resource_min_level</strong> <em>level</em> }]
+ <em>insetresource</em>} </code>...</pre>
+
+  <h3>Purpose</h3>
+  <p>Modifies resources of an existing project. </p>
+  
+  <h3>Commands</h3>
+  <pre><code><b>--mercator </b></code></pre>
+  <p><i>Optional</i>. Uses Mercator map projection for the imagery project.</p>
+  <pre><code><b>--flat </b></code></pre>
+  <p><i>Default</i>. Uses Flat map (Plate Carrée) projection.</p>
+  <pre><code><b>--maxleveloverride</b> <i>level</i></code></pre>
+  <p><i>Optional</i>.</p>
+  <pre><code><b>--no_terrain_overlay</b></code></pre>
+  <p><i>Default</i>. make this terrain project a normal project.</p>
+  <pre><code><b>--terrain_overlay</b></code></pre>
+  <p><i>Optional</i>. make this terrain project an overlay project.</p>
+  <pre><code><b>--start_level</b> <i>level</i></code></pre>
+  <p><i>Optional</i>. the level from which to start building the terrain overlay project. start_level is an even unsigned integer between 4 and 24 inclusive.</p>
+  <pre><code><b>--resource_min_level</b> <i>level</i></code></pre>
+  <p><i>Optional</i>. the threshold level that separates fill terrain from overlay terrain. resource_min_level is any integer between 4 and 24 inclusive.</p>
+
+
+  <h2>genewimageyproject</h2>
+  <pre>
+<code><b>genewimageyproject</b> [--<strong>mercator</strong> | --<strong>flat]</strong> [--<strong>historical_imagery</strong> | --<strong>no_historical_imagery</strong>]
+ -o <em>projectname</em> {[--<strong>maxleveloverride</strong> <em>level</em>]}
+ <em>insetresource</em>} </code>...</pre>
+
+  <h3>Purpose</h3>
+  <p>Modifies resources of an existing project. </p>
```

Comment:
For `genewimageryproject` the purpose should specify that it "Creates a new imagery project." instead of "Modifies resources of an existing project."